screenpipe-app-tauri/components/cli-command-dialog.tsx

```diff
@@ -138,7 +139,10 @@ export function CliCommandDialog({ settings }: CliCommandDialogProps) {
             </DialogDescription>
           </DialogHeader>
           <div className="overflow-x-auto">
-            <CodeBlock language="bash" value={generateCliCommand()} />
+            <CodeBlock
+              language={platform() === "windows" ? "powershell" : "bash"}
```

Comment:
nice!

can you do `bun tauri build`

usually you cannot use `platform()` like this - build crash

because nextjs does not know tauri APIs

so you need to put it in a function or useeffect

maybe it would make sense to create a hook because we have useeffect for this everywhere - kinda dirty

Reply:
@louis030195 done.